scissors/main.c

```diff
@@ -0,0 +1,86 @@
+#include <stdio.h>
+#include <time.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+enum {
+	NONE	= -1,
+	ROCK	= 0,
+	SCISSORS= 1,
+	PAPER	= 2,
+	OPTS_MAX
+}options;
+
+const int rules_martix[OPTS_MAX][OPTS_MAX] =
```

Comment:
All entities that are not used outside the current file must be static.

Reply:
Will fix